ui/client/components/graph/NodeDetailsModalHeader.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SvgDiv from "../SvgDiv";
+
+const HeaderType = {
+  'SUBTYPE_DOCS' : 1,
+  'SUBTYPE' : 2,
+  'DOCS': 3,
```

Comment:
Please add 'DEFAULT' and at the end in getModalHeaderType return 'DEFAULT' we should always return some type.